test/ui5-app/webapp/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>bare bones UI5 (dark 😱)</title>
+  <script id="sap-ui-bootstrap"
+          src="resources/sap-ui-core.js"
+          data-sap-ui-theme="sap_fiori_3_dark"
+          data-sap-ui-xx-cssVariables="true"
+          data-sap-ui-async="true">
+          </script>
+  <script>
+    sap.ui.require([
```

Comment:
see my suggestion via slack - use `libs` and `oninit` with callback

Reply:
done